raiden/network/proxies/secret_registry.py

```diff
@@ -256,7 +256,7 @@ def register_secret_batch(self, secrets: List[Secret]):
             # assert/revert, and the account should always be funded
             self.proxy.jsonrpc_client.check_for_insufficient_eth(
                 transaction_name="registerSecretBatch",
-                address=self.node_address,
+                _address=self.node_address,
```

Comment:
We should be using the `_address` parameter instead of:

`our_address = to_checksum_address(self.address)`

The method uses the JSONRPC client address, but we should be able to pass an address parameter to check also for LC addresses.